docker/entrypoint.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash -x
+
+# Copyright (c) 2016 Uber Technologies, Inc.
+
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+# start up cassandra
+pushd /
+./docker-entrypoint.sh cassandra
+popd
+sleep 20
+
+export HOST_IP=`hostname --ip-address`
+
+# setup schema
+sed -i.bak "s/'replication_factor':\ '5'/'replication_factor':\ '1'/g" clients/metadata/schema/metadata_keyspace.cql
+ln -sf `which cqlsh` /usr/local/bin/cqlsh
+CQLSH_HOST=$HOST_IP ./scripts/cherami-setup-schema
```

Comment:
Question was about CQLSH_HOST.. Forgot it was an env that cqlsh understands. nvm..